windows/runner/flutter_window.h

```diff
@@ -0,0 +1,39 @@
+#ifndef RUNNER_FLUTTER_WINDOW_H_
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase